AIP-7418 Remove KFP cluster create

talebzeghmi opened the pull request:
This PR removes the ability for WFSDK to create sticky long term k8s resources in the k8s cluster such as the WorkflowTemplate and CronWorkflow.  

The reason for this is that k8s garbage collection and pruning policy becomes much more difficult to manage than Gitlab projects that have environments, and PRs that get merged with attached cleanup jobs and corresponding pruning steps by label.

TODO
- [x] update the flow_tiggering_flow.py test to use kubectl to create a yaml generated workflow template to trigger instead of the previous `create` command

talebzeghmi committed:
AIP-7418 Remove KFP create

alexlatchford commented on metaflow/plugins/kfp/kfp_cli.py:
```diff
@@ -541,30 +528,7 @@ def create(
-        #  flow.schedule()
-        #  obj.echo("What will trigger execution of the workflow:", bold=True)
-        #  obj.echo(flow.trigger_explanation(), indent=True)
+        raise KfpException("create command is only supported with --yaml-only")
```
Does it make sense to swap the default of `yaml_only` to be `True` now to make this the happy path?

talebzeghmi replied:
The concern is what happens when this command would truly allow creating in the cluster?
So making it explicit that to be yaml_only is the approach.  Furthermore, this is an advanced command

alexlatchford replied:
Cool will defer to your judgement here then, thanks for the context :)

talebzeghmi committed:
AIP-7385 rename KFP to AIP plugin (#258)

* AIP-7385 rename KFP to AIP plugin

* friendly error message for "kfp run"

* AIP-7393 Add Flow labels to Workflow for ArgoUI (#259)

* AIP-7393 Add Flow labels to Workflow for ArgoUI

* PR feedback

* AIP-7726 add default cards (#260)

* AIP-7726 add default cards

* param fix

* PR feedback
- comment archive none
- add card id to support current.card.append()
- add example of notebook card & multiple cards

* aip.zillowgroup.net/aip-pod-default -> aip.zillowgroup.net/aip-wfsdk-pod

* logging.zgtools.net/index -> logging.zgtools.net/topic

* fix card pathspec for nested foreach

* fix github test.yml branches

* resources_flow.py label fixes

* PR Feedback

* update test.yml

talebzeghmi committed:
AIP-7773 argo flow trigger flow (#261)
